package-lock.json

```diff
@@ -1,25 +1,22 @@
 {
   "name": "@teachiq/eslint-config",
-  "version": "2.0.0",
+  "version": "2.0.1-beta.2",
```

Comment:
Change when releasing.
What would be the appropriate version ? Since TS was support is removed until proper implementation (better not having it than having a bad one) should this be a major ?

Reply:
I believe that the `"extends": "@teachiq"` just points to the index.js of the eslint config package. Maybe it's as simple to just duplicate these files into `javascript.js` and `typescript.js`. It will still be a breaking change, but maybe a more sane way to keep TS support.

Reply:
This way the config can be extended as `"extends": "@teachiq/javascript"` and `"extends": "@teachiq/typescript"`

Reply:
Interesting. I will give that a try :) !